common/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/common/model/IndexStatus.kt

```diff
@@ -10,6 +10,7 @@ import org.springframework.data.elasticsearch.annotations.FieldType
 import java.time.LocalDateTime
 
 const val INDEX_STATUS_ID = "STATUS"
+const val INCENTIVES_INDEX_STATUS_ID = "INCENTIVES_STATUS"
```

Comment:
Store a second document in the status index to manage the new incentives index

Reply:
Why would the prisoner / incentive indexes need managing separately? Wouldn't they come as a "set" and all be doing the same thing at the same time?

Reply:
I wasnt sure how flexible we need to be on this, e.g. is the ability to swap from the A to the B index useful for one of the indices? 🤷

Reply:
Yeah I'm just thinking it through some more... it might actually be quite useful to manage them independently. For example if one has its model changed you only need to rebuild that one. 

Or maybe it would just get too complicated - e.g. NOMIS data is GREEN, incentives is BLUE and GREEN is currently rebuilding, restricted patients is GREEN but BLUE is rebuilding. I'm lost now.